.github/workflows/CI.yml

```diff
@@ -46,92 +46,93 @@ jobs:
-          max_attempts: 3
-          retry_on: error
-          command: |
-            pnpm test.nuclide.node
-            # pnpm test.nuclide.atom
-
-      - name: Run Windows tests 👩🏾‍💻
-        if: startsWith(matrix.os,'windows')
-        continue-on-error: true # 11 of the windows tests fail
-        run: |
-          pnpm test.nuclide.node
-          # pnpm test.nuclide.atom
-
-  Lint:
-    if: "!contains(github.event.head_commit.message, '[skip ci]')"
-    runs-on: ubuntu-latest
-    env:
-      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    steps:
-      - uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-      - name: Commit lint ✨
-        uses: wagoid/commitlint-github-action@v1
-        
-      - uses: UziTech/action-setup-atom@v1
-      - name: Setup PNPM
-        uses: pnpm/action-setup@v1.2.1
-        with:
-          version: latest
-
-      - name: Install NPM dependencies
-        run: |
-          npm install
-      - name: Lint ✨
-        run: npm run lint
+  # Nuclide-Test:
+  #   if: "!contains(github.event.head_commit.message, '[skip ci]')"
+  #   runs-on: ${{ matrix.os }}
+  #   strategy:
+  #     fail-fast: false
+  #     matrix:
+  #       os:
+  #         - ubuntu-latest
+  #         - macos-latest
+  #         - windows-latest
+  #       atom_channel: [stable, beta]
+  #   steps:
+  #     - uses: actions/checkout@v2
+  #
+  #     - name: Cache node_modules
+  #       id: github-cache-conan
+  #       uses: actions/cache@v2
+  #       env:
+  #         cache-name: |
+  #           ~/.atom/packages
+  #           ~/.pnpm-store
+  #       with:
+  #         path: node_modules
+  #         key: pnpm-${{ runner.os }}-${{ matrix.atom_channel }}-${{ hashFiles('package.json') }}
+  #
+  #     - uses: UziTech/action-setup-atom@v1
+  #       with:
+  #         channel: ${{ matrix.atom_channel }}
+  #     - name: Versions
+  #       run: apm -v
+  #
+  #     - name: Setup PNPM
+  #       uses: pnpm/action-setup@v1.2.1
+  #       with:
+  #         version: latest
+  #     - run: |
+  #         pnpm install
+  #         pnpm prepare.nuclide
+  #
+  #     - name: Run Linux tests 👩🏾‍💻
+  #       if: startsWith(matrix.os,'ubuntu')
+  #       run: |
+  #         pnpm test.nuclide.node
+  #         # pnpm test.nuclide.atom
+  #
+  #     - name: Run macos tests 👩🏾‍💻
+  #       if: startsWith(matrix.os,'macos')
+  #       uses: nick-invision/retry@v2 # macOS flaky tests sometimes need retries
+  #       with:
+  #         timeout_minutes: 10
+  #         max_attempts: 3
+  #         retry_on: error
+  #         command: |
+  #           pnpm test.nuclide.node
+  #           # pnpm test.nuclide.atom
+  #
+  #     - name: Run Windows tests 👩🏾‍💻
+  #       if: startsWith(matrix.os,'windows')
+  #       continue-on-error: true # 11 of the windows tests fail
+  #       run: |
+  #         pnpm test.nuclide.node
+  #         # pnpm test.nuclide.atom
+
+  # No point in linting in CI if all lint errors are warnings
```

Comment:
In the new GitHub interface, the warnings are shown as messages in the git diff. So even if they are warnings, we can see them (especially if they are in the edited code).

Reply:
That doesn't really help since there are so many warnings they are just ignored. lol

Reply:
I think the interfaces show the warnings if they are in the same file of the edited code.

Anyhow, most of the warnings are because of Nuclide packages. I don't want to spend time on those unless for those parts that we use. e.g.: nuclide-debugger-common